wordpress-mobile/WordPress-Android
# Trash post confirm dialog

Fixes #7374 

![delete_alert](https://user-images.githubusercontent.com/6597771/37034054-c7b62602-2126-11e8-8c0a-a5980cb7cff8.gif)

**Notes**
- A slight difference in title from what the issue #7374 describes: this PR uses the Title `Delete post`  and `Delete page` for the dialog, to keep them short and according to what was already being used in the app for other dialogs.

**To test:
CASE A:**
1. go to Posts list
2. try to trash/delete a Post by tapping on the  Trash/delete action icon on the right-bottom corner of the post's card within the list.
3. see the confirmation dialog appear.

**CASE B:**
1. go to Pages list
2. try to trash/delete a Page by tapping on the  Trash/delete action icon on the right-bottom corner of the post's card within the list.
3. see the confirmation dialog appear, and the right wording ("page") is used instead of Post.
